app/dashboard/templates/shared/cart_nav.html

```diff
@@ -1,7 +1,7 @@
-<div id="cart-nav" class="dropdown">
-    <a href="/grants/cart" class="nav-link">
+<div id="cart-nav" class="nav-item dropdown">
+    <a href="/grants/cart" class="nav-link dropdown-toggle dropdown-toggle-md-no-caret">
```

Comment:
url `{% url 'grants:cart' %}` ?

Reply:
Resolved in 7cd0d2e3bf96cc58f09dafa0833650d0731f2c7b